etsin_finder/frontend/js/components/dataset/data/idaResources/downloadActions.js

```diff
@@ -100,4 +100,20 @@ const getDownloadAction = (datasetIdentifier, item, Packages, Files) => {
   return action
 }
 
+export const getAllowDownload = (DatasetQuery, restrictions) => {
+  if (
+    DatasetQuery.results?.data_catalog?.catalog_json?.identifier ===
+    'urn:nbn:fi:att:data-catalog-pas'
+  ) {
+    return false
+  }
```

Comment:
This could be computed property called isPas in DatasetQuery.

Reply:
done